### ISSUE TITLE
Public Offline Address

### ISSUE THREAD

**Issue by beam-mw** [BeamMW/roadmap#24]
When creating offline trasactions we have two options. The first one is called just 'Offline address' and it creates the set of unique vouchers each good for one offline transaction. It is describe in [this issue (https://github.com/BeamMW/roadmap/issues/3)

The second option, that is currently planned for 5.2 version, uses a single ‘offline’ address, which allows to send any amount of offline transactions to the same address. Due to details of how LelantusMW pool works, there can only be a very limited (and definitely capped) number of such addresses created for each wallet since each new address would require the wallet to rescan the pool for UTXOs sent to it. This option is less secure since it allows deanonymization by comparing offline addresses sent out to different senders and also the sender can see when the funds are extracted from the pool. It is however much simpler for the user and more consistent with how most other crypto currencies work, which is why we recommend it for specific user cases like donations and non-interactive pool payouts, rather than general confidential transactions.

**Comment by ghost** [comment]
Interesting. I see where you're coming from. I guess it is another compromise - similar to the Audit function. It is a slippery slope, however, because in the long run it all adds up. At some point (if it ever gets there) it might become possible to force beam users to use single addresses, enable the audit option (because it is there and you have nothing to hide, don't you?) and so on. Just thinking out loud.

**Comment by r3dmarmalade** [comment]
Hi,

this is just a "basic" user comment/feedback.
So recently I wanted to add more Beam to my wallet, and this is how the whole deal went:
I added funds to my exchange account, bought BTC. I then purchased BEAM, and requested cash out to my never expiring address I have setup in beam wallet. The address is whitelisted in my exchange so that I don't have to pass additional security checks (time-consuming). Then I remembered my wallet needs to be online for this to work so I startup the wallet - since it didn't run in a while it needed to update some files, and re-scan the block chain. So then the cash out failed (timed-out) on my exchange and I needed to start a new one when the wallet was running and provide the 2-way authorizations to my exchange again and do the whole thing. Overall a time-consuming process, and while I appreciate the security I get with BEAM also it promised to be simple to use and I feel its lacking in this "basic" user case.

Perhaps a compromise solution would be to allow for single offline addresses via Lelantus and then when the wallet is opened you "secure" the funds by sending them to yourself via another address.

Definitely I would like to see just purchasing BEAM and cashing out to my wallet "working" without having to start the wallet, click extra buttons, provide additional setups etc.

**Comment by ghost** [comment]
@r3dmarmalade Good point. As long as it is secure to use an offline address, it's alright. If there's a serious compromise - I don't think so. As a side note, personally, I'm pretty happy with MW's stay-online method. It took me 2-3 transactions to get used to it.

**Comment by r3dmarmalade** [comment]
@PapaKarl0 for me this is the single biggest issue BEAM has, imagine if you had an email client that required the recipient to be online when you send him messages, or a bank account that required the same thing... The fact that BEAM developers are addressing this makes me really happy ;)
